docs/best-practices/continuous-integration/bitrise.md

```diff
@@ -0,0 +1,60 @@
+# Bitrise Integration
+
+[Bitrise](https://www.bitrise.io) allows you to automatically build, test and deploy your iOS and Android apps.
+
+## How to get started?
+
+Using _fastlane_ for your Bitrise workflow is simple. Just [add the ](https://devcenter.bitrise.io/getting-started/manage-your-bitrise-workflow) [_fastlane_](https://www.bitrise.io/integrations/steps/fastlane) [step to your
+workflow](https://devcenter.bitrise.io/getting-started/manage-your-bitrise-workflow),
+after the `Git Clone` step (and any other dependency step).
+
+### _Have our Certificate and profile installer step in your workflow!_
+_You should also add/keep the `Certificate and profile installer` step in the workflow, to download your .p12 Certificates and Provisioning Profiles uploaded to [bitrise.io](https://www.bitrise.io) and to install them. **Even if you don't upload your files to** [**bitrise.io**](https://www.bitrise.io) **and instead you use a_ _fastlane_ _tool to manage your code signing files you should still keep this step in the workflow**. Read more about [iOS Code Signing using third party tools](https://devcenter.bitrise.io/ios/code-signing/#use-a-third-party-tool-to-manage-your-code-signing-files)._
+
+With adding the _fastlane_ step we ensure that you are running on the latest _fastlane_ version, as it is pre-installed on all our VMs. Inside the step you can set the _fastlane_ action and we will run it automatically every time you push a new code change.
```

Comment:
How is this related to its headline? Should this be above under the first "Getting started"?